apps.js

```diff
@@ -1,76 +1,76 @@
-let bookList = []
+let bookList = [];
 
 const saveDataLocally = (bookList) => {
-  const stringifiedBookList = JSON.stringify(bookList)
-  localStorage.setItem('bookList', stringifiedBookList)
-}
+  const stringifiedBookList = JSON.stringify(bookList);
+  localStorage.setItem('bookList', stringifiedBookList);
+};
 
 const removeBook = (index) => {
-  bookList = bookList.filter((book, ind) => ind !== index)
-}
+  bookList = bookList.filter((book, ind) => ind !== index);
+};
 
-const listShowContainer = document.querySelector('.listShow')
+const listShowContainer = document.querySelector('.listShow');
 
 const generateBooks = () => {
-  listShowContainer.innerHTML = ''
+  listShowContainer.innerHTML = '';
 
   bookList.reverse().forEach((bookObject, index) => {
-    const div = document.createElement('div')
```

Comment:
- [ ] OPTIONAL: I think the way you implemented the `reverse()` array method is not giving the intended behavior. The list of books displayed on the page is not in an ordered manner. I think it is better if you fix this issue. 

![image](https://user-images.githubusercontent.com/64894927/144091071-e0fc86d9-fb55-431d-a4f9-5f56cbf9a1eb.png)